src/Features/Core/Portable/CodeFixes/PredefinedCodeFixProviderNames.cs

```diff
@@ -36,5 +36,7 @@ internal static class PredefinedCodeFixProviderNames
         public const string Suppression = nameof(Suppression);
         public const string AddOverloads = "Add Overloads to member";
         public const string AddNew = "Add new keyword to member";
+        public const string UseImplicitTyping = "Use var";
+        public const string UseExplicitTyping = "Use typename instead of var";
```

Comment:
Yes.  That's sort of the problem.  We shoudl not have these be inconsistent.  Either:

1. The const name should match the text.  i.e. "const string UseTypeNameInsteadOfVar" or
2. The value should match the name.  i.e. "const string UseExplicitTyping = nameof(UseExplicitTyping)"